src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/ActionShuffler.kt

```diff
@@ -0,0 +1,85 @@
+package com.atlassian.performance.tools.jirasoftwareactions
+
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.action.BrowseProjectsAction
+import com.atlassian.performance.tools.jiraactions.api.action.CreateIssueAction
+import com.atlassian.performance.tools.jiraactions.api.action.SearchJqlAction
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.JqlMemory
+import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
+import com.atlassian.performance.tools.jiraactions.api.scenario.addMultiple
+import com.atlassian.performance.tools.jirasoftwareactions.api.actions.BrowseBoardsAction
+import kotlin.reflect.KClass
+
+class ActionShuffler {
```

Comment:
The action is so tightly coupled with the scenario, that I would consider making it a private class. It doesn't feet to be reused even in the internal scope.